.evergreen/config.in.yml

```diff
@@ -13,6 +13,11 @@ command_type: system
 # That roughly accounts for variable system performance for various buildvariants
 exec_timeout_secs: 2100
 
+patch_aliases:
```

Comment:
Happy to revert.  I was thinking that once this PR lands, we can set up a nightly job that runs using this alias.

I _think_ this works but I haven't tested it yet.

Reply:
Could we move this to a separate PR? Since supporting range v2 will affect our public API, I think it's best to have a clean commit users can look at. Also, I think it would make the scope of this ticket an easier task while reviewing.

Reply:
I can remove the patch alias, but we will need the other CI changes in this PR.